### DISCUSSION TITLE
Does anyone have mp4 files for the Week 7 cutscenes?

### DISCUSSION THREAD

**Discussion by realvirtu** [FunkinCrew/Funkin#2010] [Q&A] [answered]
I'm working on my FNF engine, and I was wondering if anyone has mp4 files for the Week 7 cutscenes that I could download?

**Comment by CrusherNotDrip** [comment, marked as answer]
https://drive.google.com/drive/folders/1JjOWJU65C7layvVWizfazhWIxZ1v4qf_?usp=sharing

**Reply by CrusherNotDrip** [reply]
bro this link doesnt even work what are you on

**Reply by CrusherNotDrip** [reply]
ummm yes it does my mom said so

**Reply by CrusherNotDrip** [reply]
no it DOESNT

**Reply by CrusherNotDrip** [reply]
yes does

**Reply by CrusherNotDrip** [reply]
ts pmo....
